### PR TITLE
Created 2 new playbooks for template import and Fuse Online installation for Advanced Agile Integration course

### PR DESCRIPTION
**Pull request by honghuac**
##### SUMMARY

Created 2 new playbooks for template import and Fuse Online installation for Advanced Agile Integration course.

### CODE REVIEW

File: ansible/roles/ocp-workload-aiadv-fuseonline/ilt_provision.sh [hunk 1]
```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+END_PROJECT_NUM=1
+START_PROJECT_NUM=1
+WORKLOAD="ocp-workload-fuse-ignite"
```
Comment (fridim) [inline ansible/roles/ocp-workload-aiadv-fuseonline/ilt_provision.sh]: should it be ocp-workload-aiadv-fuseonline here ?

File: ansible/roles/ocp-workload-aiadv-fuseonline/readme.adoc [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+Afterwards, execute these commands with these env variables
+
+== Execution using oc client against remote host
+
+GUID=3d5k
+
+OCP_USERNAME=`oc whoami`
+SUBDOMAIN_BASE=`oc whoami --show-server | cut -d'.' -f 2,3,4,5 | cut -d':' -f 1`
+WORKLOAD="ocp-workload-fuse-ignite"
```
Comment (fridim) [inline ansible/roles/ocp-workload-aiadv-fuseonline/readme.adoc]: same here

File: ansible/roles/ocp-workload-aiadv-templates/ilt_provision.sh [hunk 3]
```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+END_PROJECT_NUM=1
+START_PROJECT_NUM=1
+WORKLOAD="ocp-workload-fuse-ignite"
```
Comment (fridim) [inline ansible/roles/ocp-workload-aiadv-templates/ilt_provision.sh]: same here

File: ansible/roles/ocp-workload-aiadv-templates/readme.adoc [hunk 4]
```diff
@@ -0,0 +1,33 @@
+= ocp-workload-fuse-ignite
```
Comment (fridim) [inline ansible/roles/ocp-workload-aiadv-templates/readme.adoc]: same here

File: ansible/roles/ocp-workload-aiadv-templates/readme.adoc [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+Afterwards, execute these commands with these env variables
+
+== Execution using oc client against remote host
+
+GUID=3d5k
+
+OCP_USERNAME=`oc whoami`
+SUBDOMAIN_BASE=`oc whoami --show-server | cut -d'.' -f 2,3,4,5 | cut -d':' -f 1`
+WORKLOAD="ocp-workload-fuse-ignite"
```
Comment (fridim) [inline ansible/roles/ocp-workload-aiadv-templates/readme.adoc]: same here

File: ansible/roles/ocp-workload-aiadv-fuseonline/defaults/main.yml [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+#syndesisio_sa_yml:  "{{syndesisio_url}}/install/support/serviceaccount-as-oauthclient-restricted.yml"
+#syndesisio_template_yml: "{{syndesisio_url}}/install/syndesis.yml"
+#################################################################
+
+
+
+##########          supported fuse ignite       #################
+SYNDESIS_REGISTRY: registry.access.redhat.com
+SYNDESIS_REGISTRY_WEB_SUBCONTEXT: fuse7
+product_name: fuse-ignite-1.4
```
Comment (fridim) [inline ansible/roles/ocp-workload-aiadv-fuseonline/defaults/main.yml]: same question here

File: ansible/roles/ocp-workload-aiadv-templates/ilt_provision.sh (hunk 3 above)
Comment (honghuac) [reply ansible/roles/ocp-workload-aiadv-templates/ilt_provision.sh]: @fridim Thanks, addressed using this commit 20f845963ea22fe8157cb7bee7503a8f8a564a6c

File: ansible/roles/ocp-workload-aiadv-templates/readme.adoc (hunk 4 above)
Comment (honghuac) [reply ansible/roles/ocp-workload-aiadv-templates/readme.adoc]: @fridim Thanks, addressed using this commit b05b047a4ec79cdccea9a9ee7c0f2eada88d4222

File: ansible/roles/ocp-workload-aiadv-templates/readme.adoc (hunk 5 above)
Comment (honghuac) [reply ansible/roles/ocp-workload-aiadv-templates/readme.adoc]: @fridim Thanks, addressed using this commit b05b047a4ec79cdccea9a9ee7c0f2eada88d4222

File: ansible/roles/ocp-workload-aiadv-fuseonline/defaults/main.yml (hunk 6 above)
Comment (honghuac) [reply ansible/roles/ocp-workload-aiadv-fuseonline/defaults/main.yml]: @fridim Fuse Ignite (aka Fuse Online) is installed using this playbook. So the information listed is correct.

File: ansible/roles/ocp-workload-aiadv-fuseonline/readme.adoc (hunk 2 above)
Comment (honghuac) [reply ansible/roles/ocp-workload-aiadv-fuseonline/readme.adoc]: @fridim Thanks, addressed using this commit b05b047a4ec79cdccea9a9ee7c0f2eada88d4222

File: ansible/roles/ocp-workload-aiadv-fuseonline/ilt_provision.sh (hunk 1 above)
Comment (honghuac) [reply ansible/roles/ocp-workload-aiadv-fuseonline/ilt_provision.sh]: @fridim Thanks, addressed using this commit 20f8459

**Pull request merged**